create-meta/java/pom.xml

```diff
@@ -25,39 +25,38 @@
         <url>git://github.com/cucumber/create-meta-java.git</url>
     </scm>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
-            </plugin>
-        </plugins>
-    </build>
-
     <dependencies>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>messages</artifactId>
+            <version>12.1.2-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.eclipsesource.minimal-json</groupId>
+            <artifactId>minimal-json</artifactId>
+            <version>0.9.5</version>
```

Comment:
Can we avoid this dependency or only use it in test scope? When using the `mvn exec:java` plugin you can execute code in test scope. This way we don't need to include the main method in the jar that is distributed.

Reply:
The library needs to read `ciDict.json` at runtime. I was going to shade it like it's done elsewhere.

Do you have a different suggestion @mpkorstanje?

Reply:
Never-mind. I saw it used in the main method and figured that was it. Shading will have to do.